08-projects.Rmd

```diff
@@ -4349,7 +4351,9 @@ sqlite3 /class/datamine/data/lahman/lahman.db
 - The first 10 results of the query.
 ```
 
-##### 6. Find the top 10 team total of Errors between 1960 and 1970. Display their Win and Loss totals too. What is the name of the 3rd place team?
+##### 6. Find the 10 team-year combinations that have the most number of Errors (`E`) between 1960 and 1970. Display their Win and Loss counts too. What is the name of the team that appears the 3rd place in the ranking of team-year combinations?
```

Comment:
I think the previous version might be confusing in that people may try to find the team totals _over_ the decade instead of keeping each year separately. As I result, I used `SUM()` and `GROUP BY teamID`. Hence, I changed the subject from "team" to "team-year combination" - It is just an ad-hoc name, and please feel free to modify it!